### PR TITLE
Fix sort_list over list of dicts

### LINKED ISSUE #76: eos_lag_interfaces triggers TypeError error in module_utils/network/common/utils.py
##### SUMMARY

eos_lag_interfaces fails with error
ansible/module_utils/network/common/utils.py\", line 96, in sort_list\nTypeError: '<' not supported between instances of 'dict' and 'dict'\n"

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

eos_lag_interfaces
module_utils/network/common/utils.py

##### ANSIBLE VERSION

```
ansible 2.9.10
  config file = /Users/X/Repos/N/ansible/v2/ansible.cfg
  configured module search path = ['/Users/X/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/X/n/lib/python3.7/site-packages/ansible
  executable location = /Users/X/n/bin/ansible
  python version = 3.7.8 (default, Jul  4 2020, 10:17:17) [Clang 11.0.3 (clang-1103.0.32.62)]
```

##### CONFIGURATION

```
ANSIBLE_NOCOWS(/Users/X/Repos/N/ansible/v2/ansible.cfg) = True
DEFAULT_FORKS(/Users/X/Repos/N/ansible/v2/ansible.cfg) = 10
DEFAULT_HOST_LIST(/Users/X/Repos/N/ansible/v2/ansible.cfg) = ['/Users/X/Repos/N/ansible/v2/inventories/numberly.py']
DEFAULT_MANAGED_STR(/Users/X/Repos/N/ansible/v2/ansible.cfg) = This file is managed by ansible, don't make changes here - they will be overwritten.
DEFAULT_ROLES_PATH(/Users/X/Repos/N/ansible/v2/ansible.cfg) = ['/Users/X/roles', '/Users/X/Repos/N/ansible/v2/roles']
DEFAULT_TIMEOUT(/Users/X/Repos/N/ansible/v2/ansible.cfg) = 30
DEPRECATION_WARNINGS(/Users/X/Repos/N/ansible/v2/ansible.cfg) = True
HOST_KEY_CHECKING(/Users/X/Repos/N/ansible/v2/ansible.cfg) = False
INVENTORY_ENABLED(/Users/X/Repos/N/ansible/v2/ansible.cfg) = ['host_list', 'script']
RETRY_FILES_ENABLED(/Users/X/Repos/N/ansible/v2/ansible.cfg) = False
SYSTEM_WARNINGS(/Users/X/Repos/N/ansible/v2/ansible.cfg) = True
```

##### OS / ENVIRONMENT

target is Arista switch running EOS 4.23.3M

##### STEPS TO REPRODUCE

Try to confifure a LAG using eos_lag_interfaces

```
ansible_connection: network_cli
ansible_network_os: eos

    - name: Configure uplink LAG
      eos_lag_interfaces:
        config:
          - name: "Port-Channel1"
            members:
              - member: "Ethernet49/1"
                mode: on
              - member: "Ethernet50/1"
                mode: on
        state: replaced
```

##### EXPECTED RESULTS

Should configure the lag on Arista switch

##### ACTUAL RESULTS

traceback error
TypeError: '<' not supported between instances of 'dict' and 'dict'

https://gist.github.com/cedrictabary/29450577d53eab946dcf97922a13d1b9

### PR DESCRIPTION
**Pull request by Qalthos**
##### SUMMARY

Fixes #76 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

module_utils/network/common/utils

### CODE REVIEW

File: plugins/module_utils/network/common/utils.py [hunk 1]
```diff
@@ -150,7 +150,16 @@ def transform_commands(module):
 
 def sort_list(val):
     if isinstance(val, list):
-        return sorted(val)
+        try:
+            return sorted(val)
+        except TypeError:
+            # The contents are unsortable for some reason.
+            # Maybe it's a list of ditcs:
```
Comment (sc68cal) [inline plugins/module_utils/network/common/utils.py:157]:
```suggestion
            # Maybe it's a list of dicts:
```

**Comment by sc68cal** [comment]
Looks good to me, and I need this to unbreak our CI system

**Comment by Qalthos** [comment]
recheck

**Comment by sc68cal** [comment]
Will this get picked up by https://github.com/ansible-collections/arista.eos/blob/main/galaxy.yml#L5 or do we need to new release to pick this up?

**Comment by sc68cal** [comment]
@Qalthos can you cut a new release of netcommon that includes this fix? We require it for our CI system

**Pull request merged**